deployment/stacks/api.ts

```diff
@@ -546,5 +522,8 @@ export function createApiComponents(props: CreateApiComponentsProps) {
     props.iamPath
   );
 
-  return { restApiId: api.restApiId, apiGatewayRestApiUrl: api.url };
+  return {
+    restApiId: api.restApiId,
+    apiGatewayRestApiUrl: api.url.slice(0, -1),
```

Comment:
Pardon my ignorance, what does the slice part do here?

Reply:
just slices off the last character which is the slash at the end of the url.  In seds I noticed the react frontend adds the / at the beginning when its creating the URL for an API endpoint, so if send that URL as is it ends up sending // which results in an error.

Reply:
We could make this a little more self-documenting with a regex that specifies we're removing a trailing slash: `api.url.replace(/\/$/, "")`

Wait no that's way worse, never mind.

It may be worthwhile in the future to revisit _why_ our frontend is blindly appending that `/`, make it not do that, and come back to simplify this code. On the other hand, that may not be worthwhile. If it ain't broke, right?